### PR TITLE
Closes #3404: Add release notes section to PR template

### LINKED ISSUE #3404: Update PR Template with Release Notes
I want to add a section to the PR template for release notes.

The section should be at the end and contain the following information.

```
## Release Notes
<!-- Provide a summary of changes, how to use this change, and any related links. This content will be pasted in the Important notes section of the [release notes](https://github.com/az-digital/az_quickstart/releases). -->
```

### PR DESCRIPTION
**Pull request by danahertzberg**
## Description
Adds a section for Release notes in the PR template

## Related issues
Closes #3404

## How to test
Review template for changes

### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
   - [ ] Bug fix
   - [ ] Accessibility, performance, or security improvement
   - [ ] Critical institutional link or brand change
   - [ ] Adding experimental module
   - [ ] Update experimental module
- **Minor release changes**
   - [ ] New feature
   - [ ] Breaking or visual change to existing behavior
   - [ ] Upgrade experimental module to stable
   - [ ] Enable existing module by default or database update
   - [ ] Non-critical brand change
   - [ ] New internal API or API improvement with backwards compatibility
   - [ ] Risky or disruptive cleanup to comply with coding standards
   - [ ] High-risk or disruptive change (requires upgrade path, risks regression, etc.)
- **Other or unknown**
   - [x] Other or unknown

### Drupal core
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch level release (non-security bug-fix release)
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major or minor level update
- **Other or unknown**
   - [ ] Other or unknown

### Drupal contrib projects
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch or minor level update
   - [ ] Add new module
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major level update
- **Other or unknown**
   - [ ] Other or unknown

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by trackleft — APPROVED** [review]

**Review by tadean — APPROVED** [review]

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -3,6 +3,17 @@
 ## Description
 <!--- Describe your changes in detail (include keywords close/fix/resolve) -->
 
+### Release notes
+Provide a summary of changes, how to use this change, and any related links. This content will be
```
Comment (joeparsons) [inline .github/pull_request_template.md:7]: Nitpick to make it more obvious that this is optional:
```suggestion
If this change requires release notes, provide a summary of changes, how to use this change, and any related links. This content will be 
```

**Review by trackleft — APPROVED** [review]

**Review by tadean — APPROVED** [review]

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -3,6 +3,17 @@
 ## Description
 <!--- Describe your changes in detail (include keywords close/fix/resolve) -->
 
+### Release notes
```
Comment (danahertzberg) [inline .github/pull_request_template.md:6]:
```suggestion
### Release notes
<!--- This section is intended to be deleted if there are no notes. -->
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -3,6 +3,18 @@
...
+<!--- This section is intended to be deleted if there are no notes. -->
+If this change requires release notes: provide a summary of changes, how to use this change, and any related links. This content will be 
+pasted in the [release notes](https://github.com/az-digital/az_quickstart/releases). Use markdown 
+format to ensure proper pasting of information.
```
Comment (danahertzberg) [inline .github/pull_request_template.md:10]:
```suggestion
If this change requires release notes: provide a summary of changes, how to use this change, 
and any related links. This content will be pasted in the [release notes](https://github.com/az-digital/az_quickstart/releases). Use markdown format to ensure proper pasting of information.
```

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -3,6 +3,17 @@
 ## Description
 <!--- Describe your changes in detail (include keywords close/fix/resolve) -->
 
+### Release notes
+<!--- This section is intended to be deleted if there are no notes. -->
+If this change requires release notes: provide a summary of changes, how to use this change, 
+and any related links. This content will be pasted in the [release notes](https://github.com/az-digital/az_quickstart/releases). Use markdown format to ensure proper pasting of information.
```
Comment (danahertzberg) [inline .github/pull_request_template.md:9]:
```suggestion
and any related links. This content will be pasted in the [release notes](https://github.com/az-digital/az_quickstart/releases). 
Use markdown format to ensure proper pasting of information.
```

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -3,6 +3,18 @@
 ## Description
 <!--- Describe your changes in detail (include keywords close/fix/resolve) -->
 
+### Release notes
+<!--- This section is intended to be deleted if there are no notes. -->
+If this change requires release notes: provide a summary of changes, how to use this change, 
+and any related links. This content will be pasted in the [release notes](https://github.com/az-digital/az_quickstart/releases). 
+Use markdown format to ensure proper pasting of information.
```
Comment (danahertzberg) [inline .github/pull_request_template.md:10]:
```suggestion
<!--- This section is intended to be deleted if there are no notes. -->

If this change requires release notes: provide a summary of changes, how to use this change, 
and any related links. This content will be pasted in the [release notes](https://github.com/az-
digital/az_quickstart/releases). Use markdown format to ensure proper pasting of information.
```

File: .github/pull_request_template.md [hunk 6]
```diff
@@ -3,6 +3,19 @@
 ## Description
 <!--- Describe your changes in detail (include keywords close/fix/resolve) -->
 
+### Release notes
+<!--- This section is intended to be deleted if there are no notes. -->
+
+If this change requires release notes: provide a summary of changes, how to use this change, 
+and any related links. This content will be pasted in the [release notes](https://github.com/az-
+digital/az_quickstart/releases). Use markdown format to ensure proper pasting of information.
```
Comment (danahertzberg) [inline .github/pull_request_template.md:11]:
```suggestion
If this change requires release notes: provide a summary of changes, how to 
use this change, and any related links. This content will be pasted in the 
[release notes](https://github.com/az-digital/az_quickstart/releases). Use 
markdown format to ensure proper pasting of information.
```

**Review by trackleft — APPROVED** [review]

**Review by joeparsons — APPROVED** [review]

**Review by tadean — APPROVED** [review]

**Pull request merged**